server/portal/templates/includes/nav_portal.raw.html

```diff
@@ -17,6 +17,9 @@
       font-size: 1.5em;
       vertical-align: middle; /* tweaked to align better */
   }
+  .s-header.navbar {
+    --nav-padding-horz: 60px;
```

Comment:
This works locally for me. But, thinking about mobile users or people without desktop monitors, would it make more sense to do a responsive unit like `%`? Of course if this is a intentional design choice, feel free to ignore me.

Reply:
The `%` can also be unwieldy (it scales, but not consistently). I would suggest using a normalized relative unit like `rem` instead: `--nav-padding-horz: 4rem;`